solidity/dashboard/src/components/messages/LiquidityRewardsEarnedMessage.jsx

```diff
@@ -0,0 +1,32 @@
+import React from "react"
+import { Message } from "../Message"
+import * as Icons from "../Icons"
+
+const LiquidityRewardsEarnedMessage = ({
+  liquidityRewardPairName,
+  sticky,
+  messageId,
+  onMessageClose,
+}) => {
+  const icon = Icons.Rewards
+  const title = `[${liquidityRewardPairName}] You've earned rewards!`
+  const content = <a href={"/liquidity"}>View your balance</a>
```

Comment:
Let's use `Link` from `react-router-dom`. Currently, if click the link, the dapp refreshes and disconnects from the wallet.